README.md

```diff
@@ -1,2 +1,73 @@
-# autonolas-staking-programmes
-Autonolas Staking Programmes
+# Autonolas Staking Programmes
+
+## Introduction
+This repository contains the Autonolas Staking Programmes set of contracts. It is built on top of a core service staking
+functionality of [autonolas-staking-programmes](https://github.com/valory-xyz/autonolas-staking-programmes).
```

Comment:
do you mean registries here?

Reply:
Yes, good catch, it was registries before, but the renaming changed it :)